### PR TITLE
OWNERS: Add myself

### PR DESCRIPTION
**Pull request by cgwalters**
I think I have graduated to at least a yellow belt in this codebase,
and would like the ability to approve other people's PRs.

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
The last time people were added (me and jerry) it was in https://github.com/openshift/machine-config-operator/blob/0084b0e0627feae3205a9038b471ed63f383971c/OWNERS_ALIASES as opposed to the OWNERS.

**Comment by jlebon** [comment]
/lgtm

@kikisdeliveryservice the OWNERS_ALIASES just defines group ownership. The toplevel OWNERS file determines approval for all repo modifications rather than just the MCD. I'd be cool myself with having you and Jerry in there as well. I think we're all pretty good at this point at understanding how things fit together and knowing when we should let others review. :)

File: OWNERS [hunk 1]
```diff
@@ -6,4 +6,5 @@ approvers:
   - smarterclayton
   - wking
   - ashcrow
+  - walters
```
[the remark this answers is not in the document]
Comment (cgwalters) [reply OWNERS]: Hah...yes.

**Pull request merged**